src/app/components/course-list/course-list.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit } from '@angular/core';
+import { Course } from '../course-item/course-item.component';
+
+@Component({
+  selector: 'app-course-list',
+  templateUrl: './course-list.component.html',
+  styleUrls: ['./course-list.component.scss'],
+})
+export class CourseListComponent implements OnInit {
+  public courses: Course[] = [];
+
+  public onDelete(id?: string | number) {
```

Comment:
please add function return type, please check all methods in the app, and add return type there.  In this case currently we have a void type `public onDelete(id?: string | number):  void {`